app/views/widgets/addresslookup.html

```diff
@@ -9,7 +9,7 @@
 
         <input type="text" class="govuk-input govuk-!-width-one-quarter {{ "govuk-input--error" if fields.postcode.error }}" id="postcode" name="postcode" value="{{ fields.postcode.value }}">
         <input type="hidden" id="referrer" name="referrer" value="{{ step }}">
-        <input type="submit" data-prevent-double-click="true" class="govuk-button govuk-button--secondary find-address-button" formaction="/find-address" name="findaddress" value="{{ content.findAddress }}">
+        <input type="submit" data-prevent-double-click="true" class="govuk-button govuk-button--secondary find-address-button" formaction="/find-address" value="{{ content.findAddress }}">
```

Comment:
new ticket please

Reply:
That HTML property is unused, just removing it is an optimisation.